### PR TITLE
Adds sass to sandbox

### PR DESCRIPTION
**Pull request by theinterned**
# What I did

- Added sass support to the sandbox via `sass-loader` and `node-sass`. 
- Configured `node-sass` to include the `sandbox/src` directory in it's include path (so you can `@import` directly starting form that path).
- Added some examples of using scss in [sandbox/src/examples/](sandbox/src/examples/). 

# How to test

1. run `yarn hack`
1. Add a file with a `*.scss` extension.
1. Write some scss!
1. `@import 'some.scss` in your story.
1. add classes from `some.scss` to markup in your story.
1. Watch your stories display the compiled css.

Resolves NDS-162

### CODE REVIEW

**Comment by theinterned** [comment]
Note: this PR is marked WIP: work in progress. 

Specifically I branched this off from PR #5 and so this is blocked until that PR is merged.

**Review by matthewlyle — APPROVED** [review]

**Pull request merged**
